src/build.d

```diff
@@ -525,10 +525,12 @@ alias buildFrontendHeaders = makeRule!((builder, rule) {
     builder
         .name("cxx-headers")
         .description("Build the C++ frontend headers ")
-        .msg("(DC) CXX-HEADERS")
+        .msg("(DMD) CXX-HEADERS")
         .deps([dmdDefault])
         .target(env["G"].buildPath("frontend.h"))
-        .command([dmdExeFile] ~ flags["DFLAGS"] ~
+        .command([dmdExeFile] ~
+            flags["DFLAGS"]
+              .filter!(f => matchFirst(f, ctRegex!`^-(debug=|version=|I|J)`)).array ~
```

Comment:
These can simply use `startsWith` instead of a regex

Reply:
Didn't know that `startsWith` accepted multiple strings. Neat.